//Manejando error 
try {
    hello();
}catch(error){
    console.log(error);
};